source/Octopus.Tentacle.Tests.Integration/Support/ClientAndTentacleBuilder.cs

```diff
@@ -170,11 +178,11 @@ public async Task<ClientAndTentacle> Build(CancellationToken cancellationToken)
                 tentacleEndPoint,
                 server.ServerHalibutRuntime,
                 scriptObserverBackoffStrategy,
-                retryDuration,
                 tentacleClientObserver,
-                tentacleServiceDecorator);
+                tentacleServiceDecorator,
+                new RpcRetrySettings(retryDuration) { RetriesEnabled = this.retriesEnabled });
```

Comment:
Can we make the constructor require all arguments to avoid this slightly messy initialisation